packages/table/src/common/contextMenuTargetWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { ContextMenuTarget, IProps } from "@blueprintjs/core";
+import * as React from "react";
+
+export interface IContextMenuTargetWrapper extends IProps {
+    renderContextMenu: (e: React.MouseEvent<HTMLElement>) => JSX.Element;
+    style: React.CSSProperties;
+}
+
+/**
+ * Since the ContextMenuTarget uses the `onContextMenu` prop instead of event
+ * attachment API, the prop can be lost. This wrapper helps us maintain context
```

Comment:
how about saying "uses prop instead of adding event listeners" (paraphrased) for clarity?